src/components/Api.js

```diff
@@ -0,0 +1,108 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    // constructor body
+    this._baseUrl = baseUrl;
+    this._authorization = headers.authorization;
```

Comment:
Needs correction: You should store the whole headers object here, not only the authorization and reuse it in all the methods `this._headers =`

Reply:
Got it.

Reply:
Got it.

Reply:
Needs correction: Please remove `  this._authorization ` variable and use ` this._headers` instead